### PR TITLE
add: log likelihood and readme

### PR DESCRIPTION
**Pull request by VitorBSP**
Just minimal adjusts in `llerf()` to do it compatible with `optim()` and exportable.

A incipient description/readme.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+
 ```r
 devtools::install_github("https://github.com/AlissonRP/erfG")
 
 ```
+
+## Project
+
+Project developed in the course of Statistics-UFSM for the conclusion of the 
+course. Based in a arecent paper, Zea and Andrade (2021) proposed a new generator
```
Comment (AlissonRP) [inline readme.md:18]:
```suggestion
course. Based in a recent paper, Zea and Andrade (2021) proposed a new generator
```

File: readme.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+of distributions called the erf-$G$ family. Their proposal provides special 
+distributions without adding complexity to parametric spaces of resulting models.
+Empirical evidences shows that the proposed family may solve issues of flat or 
+quasi-red likelihoods in some baselines.Due to great applied potential of the
```
Comment (AlissonRP) [inline readme.md:22]:
```suggestion
quasi-red likelihoods in some baselines. Due to the great applied potential of the 
```

File: readme.md [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+
+## Project
+
+Project developed in the course of Statistics-UFSM for the conclusion of the 
+course. Based in a arecent paper, Zea and Andrade (2021) proposed a new generator
+of distributions called the erf-$G$ family. Their proposal provides special 
+distributions without adding complexity to parametric spaces of resulting models.
+Empirical evidences shows that the proposed family may solve issues of flat or 
+quasi-red likelihoods in some baselines.Due to great applied potential of the 
+new model, this package provides a computional implementatio of the erf-$G$
```
Comment (AlissonRP) [inline readme.md:23]:
```suggestion
new model, this package provides a computational implementation of the erf-$G$ 
```

File: readme.md [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
+## Project
+
+Project developed in the course of Statistics-UFSM for the conclusion of the 
+course. Based in a arecent paper, Zea and Andrade (2021) proposed a new generator
+of distributions called the erf-$G$ family. Their proposal provides special 
+distributions without adding complexity to parametric spaces of resulting models.
+Empirical evidences shows that the proposed family may solve issues of flat or 
+quasi-red likelihoods in some baselines.Due to great applied potential of the 
+new model, this package provides a computional implementatio of the erf-$G$ 
+family. The density, distribution ande quantile functions,  besides functions to
```
Comment (AlissonRP) [inline readme.md:24]:
```suggestion
family. The density, distribution and quantile functions,  besides functions to
```

**Review by AlissonRP — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (VitorBSP) [reply readme.md:18]: I just copy that part of the Profesor, I couldn't imagine there would be errors. Thanks friennd.

**Pull request merged**